packages/console/src/components/PageLayout/index.tsx

```diff
@@ -0,0 +1,15 @@
+import type { ReactNode } from 'react';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  children: ReactNode;
+};
+
+const PageLayout = ({ children }: Props) => <div className={styles.page}>{children}</div>;
+
+export const HeadLine = ({ children }: Props) => <div className={styles.headline}>{children}</div>;
+
+export const Content = ({ children }: Props) => <div className={styles.content}>{children}</div>;
```

Comment:
Is this necessary? It would be difficult for the components to override the styles on these container div elements. Can we only provide the scss and import them wherever necessary?

Reply:
Or, we should at least provide an optional `className` prop for each of these template components.